contracts/zk-shuffle/.env.example

```
@@ -0,0 +1,8 @@
+RPC_URL="https://rpc.xion-testnet-2.burnt.com:443"
+CHAIN_ID=xion-testnet-2
+WALLET1=<REPLACE_WITH_YOUR_WALLET_ADDRESS>
+TX_HASH=5A9106DC97EB091C0E3A14E4AEF193548978FD44D5201C1FC6480207D6658FB7
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TX_HASH key should go before the WALLET1 key

(UnorderedKey)